cmds/core/lockmsrs/lockmsrs.go

```diff
@@ -18,62 +18,12 @@ import (
 	"github.com/u-root/u-root/pkg/msr"
 )
 
-const (
-	msrIA32FeatureControl  msr.MSR = 0x3A  // MSR_IA32_FEATURE_CONTROL
-	msrPkgCstConfigControl msr.MSR = 0xE2  // MSR_PKG_CST_CONFIG_CONTROL
-	msrFeatureConfig       msr.MSR = 0x13C // MSR_FEATURE_CONFIG
-	msrDramPowerLimit      msr.MSR = 0x618 // MSR_DRAM_POWER_LIMIT
-	msrConfigTDPControl    msr.MSR = 0x64B // MSR_CONFIG_TDP_CONTROL
-	ia32DebugInterface     msr.MSR = 0xC80 // IA32_DEBUG_INTERFACE
+var (
+	verbose = flag.Bool("v", false, "verbose mode")
+	verify  = flag.Bool("V", false, "Verify, do not write")
+	action  = "Locking"
```

Comment:
`action` is always set to "Locking" which seems like a mistake. Should it be set to "Verify" somewhere?